data/test/scenarios/special_note_from_movetype.cfg

```diff
@@ -0,0 +1,43 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [movetype][special_note], [store_unit], [unstore_unit]
```

Comment:
There's another push queued up to remove extra whitespace here, as noted in https://github.com/wesnoth/wesnoth/pull/7160#discussion_r1030584772